tensorflow/compiler/jit/kernels/xla_ops.cc

```diff
@@ -307,11 +321,26 @@ void XlaLocalLaunchBase::Compute(OpKernelContext* ctx) {
 
   auto elapsed = env->NowMicros() - start_time;
   VLOG(2) << "Elapsed time: " << elapsed << "us";
+
+  if (cache_ == nullptr) {
+    rm_ = ctx->resource_manager();
+    rm_->LookupOrCreate<XlaConstantOutputResource>(
```

Comment:
Compiler error: ignoring return value of function declared with 'warn_unused_result' attribute [-Werror,-Wunused-result]

I think you may need a TF_RETURN_IF_ERROR macro around this call.